libs/design/src/molecules/paginator/paginator.component.scss

```diff
@@ -1,33 +1,47 @@
+@import '../../scss/daff-util';
+
 :host {
 	display: flex;
-	justify-content: center;
 }
 
 .daff-paginator {
-
-	&__ellipsis {
-		margin-top: 13px;
-	}
-
-	&__prev {
+	&__navigation {
+		@include clickable();
 		display: flex;
-		margin-right: 20px;
 		align-items: center;
-		width: 50px;
+		appearance: none;
+		background: transparent;
+		border-radius: 3px;
+		border: 0;
+		line-height: 1em;
+		padding: 5px 10px;
+
+		&.previous {
+			fa-icon {
+				margin-right: 10px;
+			}
+		}
 
-		& fa-icon {
-			margin-right: 5px;
+		&.next {
```

Comment:
I think this should be bem style, right?

Reply:
fixed in 3c42197